spring-integration-core/src/main/java/org/springframework/integration/support/management/IntegrationManagementConfigurer.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.integration.support.management;
+
+import java.util.Map;
+
+import org.springframework.beans.BeansException;
+import org.springframework.beans.factory.SmartInitializingSingleton;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.ApplicationContextAware;
+
+import reactor.core.support.Assert;
+
+/**
+ * Configures beans that implement {@link IntegrationManagement}.
+ *
+ * TODO: This class will be expanded by INT-3755/3756.
+ *
+ * @author Gary Russell
+ * @since 4.2
+ *
+ */
+public class IntegrationManagementConfigurer implements SmartInitializingSingleton, ApplicationContextAware {
```

Comment:
I don't see how this component is used from Framework.
I think we can expose all options from here to the `integration.properties` and populate a bean for this automatically.